apps/src/componentLibrary/simpleDropdown/SimpleDropdown.story.tsx

```diff
@@ -100,13 +101,34 @@ DisabledDropdown.args = {
     {value: 'option-1', text: 'Option 1'},
     {value: 'option-2', text: 'Option 2'},
   ],
+  itemGroups: [],
   selectedValue: 'option-1',
   labelText: 'Disabled Dropdown',
   onChange: args => console.log(args),
   disabled: true,
   size: 'm',
 };
 
+export const GroupedOptionsDropdown = SingleTemplate.bind({});
+GroupedOptionsDropdown.args = {
+  name: 'grouped-options-dropdown',
+  itemGroups: [
```

Comment:
Interesting, aren't we having an error here about `items` being a required prop here?

Reply:
Funnily enough, we are not. It complains about not having `itemGroups` when not specified, but doesn't for `items`.